modules/ROOT/pages/assigning-permissions.adoc

```diff
@@ -0,0 +1,430 @@
+//. From the *Select permissions to add for this user* search box, enter *API Experience Hub Portal Amin* and select the checkbox.
+//. Click the blue plus icon.
+
+== Grant the Exchange Viewer Permission
+
+Administrators can only add APIs for which they have the Exchange Viewer permission. All administrators must have the Exchange Viewer permission to add APIs to the portal.
+
+When APIs are added to the portal, all administrators are assigned automatically the Exchange Viewer permission for the APIs added in the portal. 
+
+To grant the Exchange Viewer permission to a user:
+
+. From *Access Management*, click *Users*.
+. Click the username of the user that you want to add the Exchange Viewers permission. 
+. Select *Permissions* and click *Add Permissions*.
+. From the *Select permissions to add for this user* search box, enter *Exchange Viewers* and select the checkbox.
+. Click the blue plus icon.
+
+== Create the AEH_Anypoint Permission Set With the Minimum Permissions for the Salesforce User To Perform the Linking Process 
+
+//Apply the Principle of Least Privilege for Your Salesforce Organization
+A Salesforce user with the System Administrator profile is required to connect the Salesforce Experience Cloud and link the Salesforce organization with the Anypoint Platform organization. The System Administrator profile may have more permissions than necessary. For users with an existing Salesforce organization and potentially sensitive data, ensure the security of that data by following the principle of least privilege. Use the AEH_Anypoint permission set to configure only those permissions for the user who performs the linking process. The Salesforce user with the System Administrator profile creates the AEH_Anypoint permission set with all the necessary permissions and assigns the permission set to a Salesforce user. You can create the permission set manually in Salesforce or deploy it via your CI/CD pipeline using the *AEH_Anypoint.permissionset-meta.xml* metadata file. 
+
+The permissions for the System Admin can be downgraded with the permissions set before or after there is a connection to Salesforce. If the Salesforce user is the same user who created the connection, then that user can be assigned to the AEH_Anypoint permission set without disconnecting the portal and creating the connection again. If you want to assign the permission set to another Salesforce user who did not create the connection, the portal must be disconnected and created again. For example, the original Salesforce user who connected to Salesforce left the company.
+
+Perform the following steps to manually create the permission set, add the necessary permissions, and assign a user to the permission set:
+
+. Create the AEH_Anypoint permission set:
++
+.. Log in to Salesforce as the System Administrator.
+.. In *Setup*, enter `permission sets` in the Quick Find box and click *Permission Sets*.
+.. Click *Add*.
```

Comment:
Any way to collapse some of these steps?

Enter `AEH_Anypoint` in both the *Label* and *API Name* fields and click *Save*.
...click *System Perminssions*, and then click *Edit*.